### DISCUSSION TITLE
Is it possible to adjust the IOPS for the GitHub Action Runners?

### DISCUSSION THREAD

**Discussion by anth1y** [ubicloud/ubicloud#1485] [Q&A]
We are new to using ubicloud, and we are fans. Thank you for providing this platform. 
We currently have an action build (a straight line build) that builds a VM using packer and ansible. Our recent run was on `ubicloud-standard-16-ubuntu-2204` which took 3.5hrs to build our vm. Is there a way to tune the IOPS on ubicloud's runners? I'm more than willing to open a PR and work on this as well.

thank you in advance

**Comment by enescakir** [comment]
Hi @anth1y,
Thank you for your kind words. Currently, there's no way to adjust IOPS for individual runners. However, I believe our IOPS numbers are satisfactory since we use local disks. The issue might be a bottleneck caused by another resource.
Are your builds consistently slow, or do only certain runs slow? I check some metrics on our side.
 
Can you send your workflow run details to [support@ubicloud.com](mailto:support@ubicloud.com) for further investigation. You can find the run ID in the URL after opening a specific run's details, or you can share the runner ID found in the `Set up job` section.

**Reply by anth1y** [reply]
Thanks for getting back to us. The builds are unfortunately consistently slow...we had previously tried to use github's larger runner ` ubuntu-22.04-cores-8-containers` which took 2hr 15min and another run in 2hr 9min with the same exact build.
